app/lib/features/chat_ng/providers/notifiers/chat_editor_notifier.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:acter/features/chat_ng/models/chat_editor_state.dart';
+import 'package:acter_flutter_sdk/acter_flutter_sdk_ffi.dart'
+    show RoomEventItem;
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+class ChatEditorNotifier extends AutoDisposeNotifier<ChatEditorState> {
```

Comment:
A new notifier for ```ChatEditorState``` as opposed to ```ChatInputState```. But the benefit here is this new notifier uses ```Notifier``` (newer API) from ```StateNotifier``` migration.